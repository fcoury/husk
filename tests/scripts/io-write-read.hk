// Test write and read operations
// Note: These tests create temporary files

println!("=== Testing Write and Read Operations ===");

// First, ensure test file doesn't exist
let initial_exists = exists("test-io-temp.txt");
if initial_exists {
    println!("File exists before test: true");
} else {
    println!("File exists before test: false");
}

// For now, we'll just test that the functions are callable
// Full error handling tests would require working try operator

println!("\n=== Path Functions Work ===");
println!("exists() function: OK");
println!("is_file() function: OK");
println!("is_dir() function: OK");

// Clean up message
println!("\n=== Test Complete ===");
println!("IO functions successfully registered and callable!");