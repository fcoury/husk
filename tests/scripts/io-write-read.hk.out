=== Testing Write and Read Operations ===
File exists before test: false

=== Path Functions Work ===
exists() function: OK
is_file() function: OK
is_dir() function: OK

=== Test Complete ===
IO functions successfully registered and callable!
